stablehlo/reference/Scope.cpp

```diff
@@ -35,13 +35,15 @@ void Scope::add(Value ssaValue, Tensor runtimeValue) {
   stack_frame_[ssaValue] = runtimeValue;
 }
 
-void Scope::add(ValueRange ssaValues, ArrayRef<Tensor> runtimeValues) {
+void Scope::add(ValueRange ssaValues,
+                ArrayRef<InterpreterValue> runtimeValues) {
   assert(ssaValues.size() == runtimeValues.size());
+  if (runtimeValues.empty()) return;
```

Comment:
Do we need this extra code? If runtimeValues are empty, then zip will be empty, and the loop will have zero iterations, pretty much achieving the same result.